protos/storage.proto

```diff
@@ -94,18 +94,18 @@ message WriteResponse {
 }
 
 message QueryRequest {
-  repeated string metrics = 1;
+  repeated string tables = 1;
   string ql = 2;
 }
 
 message QueryResponse {
   common.ResponseHeader header = 1;
-  enum SchemaType {
-    AVRO = 0;
-    JSON = 1;
+  uint32 affected_rows = 2;
+  oneof rowsPayload {
+    ArrowPayload arrowPayload = 3;
   }
```

Comment:
The style should be snake case:
```suggestion
  oneof rows_payload {
    ArrowRowsPayload payload = 3;
  }
```
And ArrowPayload seems not a good name, and I'm in favor of `ArrowRowsPayload`.